src/communication/bolt/v1/states/handlers.hpp

```diff
@@ -465,4 +465,10 @@ State HandleRoute(TSession &session, const Marker marker) {
   }
   return State::Error;
 }
+
+template <typename TSession>
+State HandleLogOff(TSession &session) {
```

Comment:
Yeah, or we could just remove it completely. I wasn't sure what will end up in the handler, so I left it just in case. But it seems like it's not needed.